app/assets/stylesheets/application.scss

```diff
@@ -48,3 +49,7 @@ a:hover {
     }
   }
 }
+
+table, thead, th, td, tr {
+  border: 2px solid black;
```

Comment:
I don't think this looks very good!

You should be able to make your table look nice by using bootstrap classes - https://getbootstrap.com/docs/4.3/content/tables/